accounts/tests.py

```diff
@@ -1,3 +1,30 @@
-# from django.test import TestCase
+import pytest
+from django.contrib.auth.models import User
+from .models import UserAccount
 
-# Create your tests here.
+
+# test account to reduce redundancy
+@pytest.fixture
+def test_account(db):
+    username = 'testuser'
+    password = 'lvkmdgv32fdbf1b'
+    email = 'testuser@test.com'
+    new_user = User.objects.create_user(
+        username=username, password=password, email=email
+    )
+    return new_user
+
+
+# Create a user and check if a UserAccount instance was created
```

Comment:
This docstring (comment) should be within the function
```
def some_function():
    """This is an example function with docstring"""
```

You can read more about docstring on https://www.python.org/dev/peps/pep-0257/#one-line-docstrings